oscarlab/graphene
# [LibOS] Rename `sys/shim_fs.c` to `sys/shim_file.c`

## Description of the changes 

The file name is the same as `fs/shim_fs.c`, which makes navigation harder. The name is also confusing because it's unjustified: this file only implements miscellaneous system calls related to files, and has no relation to `shim_fs.h`.

## How to test this PR? 

If it compiles, there should be no change in behavior.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/2399)
